pkg/virt-controller/watch/node.go

```diff
@@ -59,7 +59,7 @@ func NewNodeController(clientset kubecli.KubevirtClient, nodeInformer cache.Shar
 
 	c.vmiInformer.AddEventHandler(cache.ResourceEventHandlerFuncs{
 		AddFunc:    c.addVirtualMachine,
-		DeleteFunc: func(_ interface{}) {}, // nothing to do
+		DeleteFunc: func(_ interface{}) { /* nothing to do */ },
```

Comment:
This is obviously a no-op function, why do we need a comment? I think it needs to be removed and not redecorated differently.

Reply:
For some reason SonarCloud raised a code smell issue because a no-op function is defined without a comment stating that it doesn't do anything. Such comment exists, but it wasn't inside the function body so SonarCloud didn't notice it. This small change is only to remove SonarCloud's code smell issue.

Reply:
Thank you for the explanation. For future us, a dedicated commit with this explanation will tell us not to remove it (or at least why we added it).

Reply:
Sure.
Also, since I don't know how to comment on your main comment (where you wrote about separating to different commits and so on) I'll write here that I totally agree.
In the future I will separate to different commits. I also should have write in commit description that changes are not logical and are just to clean issues. Links to specific code-smell indication also would have helped.

Thanks for all of the feedback!

Reply:
> In the future I will separate to different commits.

I would suggest to do this in this PR as well. But it is indeed not relevant to this thread, lets discuss it on the root comments.